Atomic/run.py

```diff
@@ -38,6 +39,17 @@ def run(self):
             args = self._get_args("RUN")
             if args:
                 args += self.command
+                opts_file = self._get_args("RUN_OPTS_FILE")
+                if opts_file:
+                    opts_file = self.sub_env_strings("".join(opts_file))
+                    if opts_file.startswith("/"):
+                        try:
+                            run_opts = open(opts_file, "r").read()
+                            os.environ["RUN_OPTS"] = run_opts
```

Comment:
> #521

The https://github.com/projectatomic/atomic/pull/521 as of ac83a85115cc35f467b8c35802c3be0517ab1877 seems to break the tests.